datashare-tasks/src/test/java/org/icij/datashare/asynctasks/bus/amqp/AmqpServerRuleTest.java

```diff
@@ -1,8 +1,8 @@
-package org.icij.datashare.com.bus.amqp;
+package org.icij.datashare.asynctasks.bus.amqp;
 
 import com.rabbitmq.client.Connection;
 import com.rabbitmq.client.ConnectionFactory;
-import org.icij.datashare.com.bus.amqp.AmqpServerRule;
+import org.fest.assertions.Fail;
```

Comment:
remark : it looks like there is 2 assertion frameworks. It could become a challenge to maintain at some point.